UI/MauiEmbedding/ArcGisApp/ReadMe.md

```diff
@@ -0,0 +1,21 @@
+# .NET MAUI Embedding Sample App - ArcGIS Maps SDK for .NET
+
+This sample app embeds the [ArcGIS Maps SDK for .NET](https://developers.arcgis.com/net/) in an Uno Platform application.
+
+For more information on how to use the controls from EsriMaps in an Uno Platform application via .NET MAUI Embedding, please visit [our documentation here](https://github.com/unoplatform/uno.extensions/blob/main/doc/Overview/Maui/ThirdParty-EsriMaps.md).
```

Comment:
@jeromelaban can we get an aka link for this

Reply:
Alternatively these links should point to the published documentation, not to the markdown in Github

Reply:
yes will check this, thanks @nickrandolph putting this one as a placeholder until it was published on the uno.extensions side

Reply:
links are now published @nickrandolph, I created all the related aka links. They are now updated, thanks